src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/HelpWithFees.java

```diff
@@ -14,4 +14,5 @@ public class HelpWithFees {
 
     private YesOrNo helpWithFee;
     private String helpWithFeesReferenceNumber;
+    private YesOrNo invalidHelpWithFee;
```

Comment:
It will be better to store this value in the HelpWithFeesDetails object created for this journey.

Reply:
done